README.md

```diff
@@ -399,6 +402,7 @@ Errors in Data](https://journal.r-project.org/archive/2010-2/RJournal_2010-2_Sar
   - [textblob](https://textblob.readthedocs.io/en/dev/): Simplified Text processing.
     - [Natural Language Basics with TextBlob](http://rwet.decontextualize.com/book/textblob/)
   - [Gensim](https://radimrehurek.com/gensim/): Topic Modeling for humans.
+  - [Pattern.en](https://www.clips.uantwerpen.be/pages/pattern-en): The pattern.en module contains a fast part-of-speech tagger for English, sentiment analysis, tools for English verb conjugation and noun singularization & pluralization, and a WordNet interface.
```

Comment:
This is great. For consistency, please remove the text "The pattern.en module contains"

Reply:
Removed the text